packages/notifi-react-card/lib/components/subscription/EventTypeBroadcastRow.tsx

```diff
@@ -41,6 +53,10 @@ export const EventTypeBroadcastRow: React.FC<EventTypeBroadcastRowProps> = ({
   const { instantSubscribe } = useNotifiSubscribe({
     targetGroupName: 'Default',
   });
+  const {
+    canary: { isActive: isCanaryActive, frontendClient },
```

Comment:
would it be easier to handle which client being used  from the notifiClientContext?